taxi/forms.py

```diff
@@ -0,0 +1,58 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+
+from taxi.models import Car, Driver, Manufacturer
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
+        widget=forms.CheckboxSelectMultiple,
+        required=False
+    )
+
+    class Meta:
+        model = Car
+        fields = "__all__"
+
+
+class DriverForm(forms.ModelForm):
+    LICENSE_LEN = 8
+    LETTERS_LICENSE = 3
+    DIGITS_LICENSE = 5
+
+    class Meta:
+        model = Driver
+        fields = ("username", "password", "first_name",
+                  "last_name", "license_number")
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if len(license_number) != DriverForm.LICENSE_LEN:
+            raise ValidationError(f"License number should be"
+                                  f" {DriverForm.LICENSE_LEN} symbols!")
+        elif not license_number[:DriverForm.LETTERS_LICENSE].isupper():
```

Comment:
You don't have to use `elif` if your previous statement uses `return/break/continue/raise`

Reply:
Car detail page, with Car assigned to the driver and not assigned."this scrennshots added in top

Reply:
Ou, missed this)